### DISCUSSION TITLE
_eq: null filter not work in directus_collections table

### DISCUSSION THREAD

**Discussion by zippax** [directus/directus#14989] [Archive]
### Describe the Bug

unable to use `_eq: null ` in directus_collections table
Trying to show list of the collections not set to any group
```
service
    .readByQuery({
	filter: {hidden: false, group: {_eq: null}},
	fields: ['*']
    })

```

if I use `_neq: null` it works properly without any issue. and showed me all collections belongs to group

### To Reproduce

in the response it show that collection has group with null value
![Screen Shot 2022-08-09 at 10 05 29 PM](https://user-images.githubusercontent.com/3060098/183741033-832d8c10-a4e1-4baf-ab1f-b2b7d5c0fe6e.png)

### Errors Shown

no error showing in console

### What version of Directus are you using?

Directus 9.15.1

### What version of Node.js are you using?

16.7.0

### What database are you using?

mysql

### What browser are you using?

Postman

### How are you deploying Directus?

locally

**Comment by licitdev** [comment]
For null check, please use the `_null` filter operator instead. 
Ref: https://docs.directus.io/reference/filter-rules/#filter-operators

On a side note, your filter `hidden: false` is in an invalid syntax.
Ref: https://docs.directus.io/reference/filter-rules/#syntax.

**Reply by zippax** [reply]
`filter: {group: _null}` it showing me a message: `_null is not defined`

**Comment by zippax** [comment]
find the solution, I have to set `_null: true`
`filter: {hidden: false, group: {_null: true}}`
